### PR TITLE
Add compute mailjet sample

### PR DESCRIPTION
**Pull request by shun-fan**
Update maven gcloud plugin version for managed vm samples
Update master pom

### CODE REVIEW

File: compute/mailjet/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Running on Compute Engine
+
+To run the sample, you will need to do the following:
+
+1. [Create a Mailjet Account](https://app.mailjet.com/signup).
+1. Create a compute instance on the Google Cloud Platform Developer's Console
+1. SSH into the instance you created
+1. Update packages and install required packages
+
+    `sudo apt-get update && sudo apt-get install git-core openjdk-8-jdk maven`
```
Comment (lesv) [inline compute/mailjet/README.md]: Should we give them a script that does this?  So they can create their instance and set it up?

**Comment by lesv** [comment]
After you get travis happy, I'm happy.  Consider the script, but ok w/o.

**Comment by shun-fan** [comment]
Do we need a script for such a small amount of instructions?

**Comment by lesv** [comment]
I like to as it keeps them from ssh to the instance.  It also encourages them to think doing lots of stuff.

**Pull request merged**